### PR TITLE
[MTDSA-6933] - adjusting minimum tax year

### PR DESCRIPTION
**Pull request by DreamcastKnight**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -60,5 +60,6 @@ class AppConfigImpl @Inject()(config: ServicesConfig, configuration: Configurati
 
 trait FixedConfig {
   // Minimum tax year for MTD
-  val minimumTaxYear = 2019
+
+  val minimumTaxYear = 2022
```
Comment (Mbrookesy) [inline app/config/AppConfig.scala:64]: You need to move the minimumTaxYear from fixed config to the overridable appconfig section.

File: it/v1/endpoints/SampleControllerISpec.scala [hunk 2]
```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
```
Comment (Mbrookesy) [inline it/v1/endpoints/SampleControllerISpec.scala:2]: Why is there a sample controller here?

File: test/v1/controllers/requestParsers/validators/AmendOtherDeductionsValidatorSpec.scala [hunk 3]
```diff
@@ -29,7 +29,7 @@ import v1.models.request.amendOtherDeductions.AmendOtherDeductionsRawData
 class AmendOtherDeductionsValidatorSpec @Inject()(implicit appConfig: AppConfig, currentTaxYear: CurrentTaxYear) extends UnitSpec {
 
   private val validNino = "AA123456A"
-  private val validTaxYear = "2019-20"
+  private val validTaxYear = "201-22"
```
Comment (Mbrookesy) [inline test/v1/controllers/requestParsers/validators/AmendOtherDeductionsValidatorSpec.scala:32]: taxYear is mispelt

**Review by Mbrookesy — CHANGES_REQUESTED** [review]

File: test/v1/controllers/requestParsers/validators/AmendOtherDeductionsValidatorSpec.scala [hunk 4]
```diff
@@ -29,7 +29,7 @@ import v1.models.request.amendOtherDeductions.AmendOtherDeductionsRawData
 class AmendOtherDeductionsValidatorSpec @Inject()(implicit appConfig: AppConfig, currentTaxYear: CurrentTaxYear) extends UnitSpec {
```
Comment (james-work-account) [inline test/v1/controllers/requestParsers/validators/AmendOtherDeductionsValidatorSpec.scala:29]: I know you didn't change this but the Inject stuff is unnecessary (also remove any unused imports please).

```suggestion
class AmendOtherDeductionsValidatorSpec extends UnitSpec {
```

**Review by james-work-account — CHANGES_REQUESTED** [review]
Also `currentTaxYear` isn't used in MtdTaxYearValidation - again I know this wasn't put in by you but it'd be good to take it out just to get rid of the compilation warnings.

**Review by james-work-account — APPROVED** [review]

**Pull request merged**